docs/user-guides/cluster-role-binding.md

```diff
@@ -0,0 +1,131 @@
+# Cluster Role Configuration
+
+The cluster role contains rules that represent a set of permission to grant access to a specific target depending on the binding rule.
+
+This page describes why we need a cluster role for Vald and how to configure it.
+
+## What are cluster role and cluster role binding for the Vald cluster?
+
+In Vald, the index is distributed across the Kubernetes cluster depending on the resource usage of the node, it requires settings to grant permission to a specific role to retrieve cluster information on Kubernetes.
+
+By default, the cluster role settings are deployed automatically when using helm to deploy.
+
+The following manifest will be deployed.
+
+- [clusterrole.yaml](https://github.com/vdaas/vald/blob/main/k8s/discoverer/clusterrole.yaml)
+- [clusterrolebinding.yaml](https://github.com/vdaas/vald/blob/main/k8s/discoverer/clusterrolebinding.yaml)
+
+These configurations allow the service account `discoverer` to access different resources in the Kubernetes cluster.
+
+### Cluster role settings for Vald Discoverer
```

Comment:
It is little bit logical scheme, why this page shows only discoverer.
It should be mentioned why we describe only discoverer.
User may not understand the flow of the story.